packages/react-components/src/components/Picker/Trigger.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import cx from 'clsx';
+import {
+  ChevronDown,
+  ChevronUp,
+  Close,
+} from '@livechat/design-system-icons/react/material';
+import { Icon, IconSizeName, IconTypeName } from '../Icon';
+import styles from './Trigger.module.scss';
+
+const baseClass = 'picker-trigger';
+
+export const enum TriggerSize {
+  Compact = 'compact',
+  Medium = 'medium',
+  Large = 'large',
+}
```

Comment:
I would change it to:

```
type TriggerSize = 'compact' | 'medium' | 'large'
```

Using enums have some disadvantages...and with plain string union we simplify the API. Instead of importing enum we could write something like:

```
<Trigger size="compact" />
```

instead of

```
import { TriggerSize, Trigger } ...
<Triger size={TriggerSize.Compact} />
```

And using string union will fix the problem of casting:

```
styles[`${baseClass}--${size as string}`],
```

WDYT?

Reply:
👍